src/components/Input/Input.scss

```diff
@@ -4,6 +4,7 @@
 }
 
 .input-container {
+  margin: 25px 0;
```

Comment:
This feels a little too big. I think 15 looks good, but even 20 looks a little better.